addons/operating-system-manager/README.md

```diff
@@ -1,27 +1,3 @@
 # Operating-system-manager addon
 
-Operating system manager can be used to create and manage worker node configurations in a kubernetes cluster.
-
-Once enabled, this addon will take over the responsibility for managing user-data for worker machines in the cluster.
-
-**Note:** Existing worker machines will not be rotated if this addon is enabled after cluster creation.
-
-## Enabling the addon
```

Comment:
I would leave this, just mention that the addon is controlled via API and give an example.